docker/development/Dockerfile

```diff
@@ -106,7 +106,7 @@ RUN git clone https://github.com/geographiclib/geographiclib /tmp/geographiclib
 
 ## Open Space Toolkit ▸ Core
 
-ARG OSTK_CORE_VERSION="0.6.0"
+ARG OSTK_CORE_VERSION="0.7.0"
```

Comment:
All good with using this (it will need get propagated to downstream repos ⚠️). Also let's bump `requirements.py` to have them in sync if all tested

Reply:
Hmmm... good point. Wasn't this was the thing that was breaking Requesting pipeline? This is not actually needed, I just wanted the stack traces for development, so if it will break I can revert to 0.6